docs/getting-started/zowe-security-authentication.md

```diff
@@ -0,0 +1,48 @@
+* [Multi-factor authentication (MFA)](#multi-factor-authentication-mfa)
+* [Certificate Authority Advanced Authentication Mainframe (CA AAM)](#certificate-authority-advanced-authentication-mainframe-ca-aam)
+
+## Authentication with JSON Web Tokens(JWT)
+
+When the user successfully authenticates with the API ML, the client receives a JWT token in exchange. This token can be used by the client to access REST services behind the API ML Gateway and also for subsequent user authentication. The access JWT Token is signed with the private key that is configured in the Zowe Identity Provider's certificate store, be it keystore or keyring.
+
+To utilize [Single-Sign-On (SSO)](../user-guide/systemrequirements-zos#single-sign-on-sso), the Zowe API ML client needs to provide an access token to API services in the form of the cookie `apimlAuthenticationToken`, or in the `Authorization: Bearer` HTTP header as described [here](https://github.com/zowe/sample-spring-boot-api-service/blob/master/zowe-rest-api-sample-spring/docs/api-client-authentication.md#authenticated-request).
+
+## Authentication with client certificates
+
+If the keyring or a truststore contains at least one valid certificate authority (CA) other than the CA of the API ML, it is possible to use client certificates issued by this CA to authenticate to the API ML.
+
+For details, see the [Authentication for API ML services documentation](../extend/extend-apiml/authentication-for-apiml-services#authentication-parameters).
+
+## Authentication with Personal Access Token (PAT)
+
+A Personal Access Token (PAT) is a specific scoped JWT with configurable validity duration. PAT authentication method is an alternative to using client certificate for authentication. It is disabled by default. To enable this functionality, see [the configuration documentation](../user-guide/api-mediation/api-gateway-configuration#personal-access-token ).
+
+**Benefits**
+
+- Long-lived. The maximum validity is 90 days.
+- Scoped. Users are required to provide a scope. It is only valid for the specified services.
+- Secure. If a security breech is suspected, the security administrator can invalidate all the tokens based on criteria as established by rules.
+
+For more information about PAT, see [the Personal Access Token documentation](../user-guide/api-mediation/api-mediation-personal-access-token).
+
+## Authentication with SAF Identity Tokens
+The SAF Authentication Provider allows the API Gateway to authenticate the user directly with the z/OS SAF provider that is installed on the system.
+
+For more information about configuring the token, see the [Configure signed SAF Identity tokens (IDT) documentation](../user-guide/configure-zos-system#configure-signed-saf-identity-tokens-idt).
+
+## Multi-factor authentication (MFA)
+
+Multi-factor authentication is provided by third-party products which Zowe is compatible with. The following are known to work:
+
```

Comment:
This list of functional MFA options needs to be added or the statement should be removed.

Reply:
Missed that. Will add them.